modular_skyrat/master_files/code/modules/mob/living/sillicon/robot.dm

```diff
@@ -1,17 +1,20 @@
 ////
 //	Typing indicator icon change
-/mob/living/silicon/robot/set_typing_indicator(state)
-	var/mutable_appearance/indicator = mutable_appearance('modular_skyrat/modules/indicators/icons/typing_indicator.dmi', "borg0", FLY_LAYER)
+/mob/living/silicon/robot/set_typing_indicator(state, emote)
+	var/mutable_appearance/type_indicator = mutable_appearance('modular_skyrat/modules/indicators/icons/typing_indicator.dmi', "borg0", FLY_LAYER)
```

Comment:
Because this never changes this should be a static var to prevent constant image manipulation. 
(Micro optimization)

Reply:
Good point.